requirements.txt

```diff
@@ -15,5 +15,6 @@ git+https://github.com/iterait/apistrap.git
 # testing
 pytest
 pytest-mock
```

Comment:
Flask_CORS and Flask are still required?|

Reply:
touché